cypress/integration/rancid_tomatillos_tests.js

```diff
@@ -15,15 +18,15 @@ describe('Rancid Tomatillos', () => {
     });
 
     it('should show an informative Unprocessable Entity error message', () => {
+      cy.get('h2').contains('Loading...')
       cy.intercept({
           method: 'GET',
           url: 'https://rancid-tomatillos.herokuapp.com/api/v2/movies'
         },
         {
           statusCode: 422,
         })
-        cy.visit('http://localhost:3000')
-        .get('h2').contains('This is a 400 error message on the Movie Library Page')
+        cy.wait(5000).get('h2').contains('This is a 400 error message on the Movie Library Page')
     });
```

Comment:
This error message now contains a page refresh button that we could also incorporate into this test